fn.rs

```diff
@@ -0,0 +1,8 @@
+
+fn foo(x:  fn(&u8, &u8), y: Vec<&u8>, z: &u8) {
```

Comment:
Did you accidentially left this file in the root dir?

Reply:
Oh yeah. I'm open a PR for that or I have an ongoing PR, I'll make sure to delete that.

Reply:
Fixed in #44549. Thanks!